testlib/build.gradle

```diff
@@ -31,3 +31,12 @@ dependencies {
     api deps.test.mockito
     api deps.test.hamcrest
 }
+
+protobuf {
```

Comment:
This section requires commenting. I am not sure why this change is required to address the issue.

And please kill the empty line.